Tools/WinMLRunner/OutputHelper.h

```diff
@@ -14,7 +15,15 @@ using namespace Windows::Storage::Streams;
 class OutputHelper
 {
 public:
-    OutputHelper(bool silent) : m_silent(silent) {}
+    OutputHelper(int num, bool silent)
```

Comment:
Can we make the naming of "num" to be more specific?

Reply:
Okay.